app/views/pages/dashboard_blocks/_cars.html.haml

```diff
@@ -1,3 +1,10 @@
-%ul.list-group
```

Comment:
It's not a table. It's not necessary to change it. Please don't change this block.